components/desktop/librecad/Makefile

```diff
@@ -0,0 +1,114 @@
+COMPONENT_CLASSIFICATION = Applications/Graphics and Imaging
+
+COMPONENT_REVISION = $(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+IPS_COMPONENT_VERSION = $(COMPONENT_VERSION)
+
+$(SOURCE_DIR)/.downloaded: $(ARCHIVES:%=$(USERLAND_ARCHIVES)%)
+	@echo "target .download"
+	[ -d $(SOURCE_DIR) ] || \
+	$(GIT) clone -b $(GIT_TAG) --single-branch --depth 1 $(GIT_REPO) $(SOURCE_DIR) && \
+	cd $(SOURCE_DIR) && $(GIT) log -1 --format=%H > .downloaded
+
+include ../../../make-rules/prep.mk
+# include ../../../make-rules/cmake.mk
+include ../../../make-rules/justmake.mk
+
+# publish: update-publish
+include ../../../make-rules/ips.mk
+
+GIT = git
+GIT_REPO = https://github.com/LibreCAD/LibreCAD.git
+GIT_TAG = $(COMPONENT_VERSION)
+GIT_BRANCH = master
+
+tolower = $(shell echo $(1) | tr '[A-Z]' '[a-z]')
+COMPONENT_PRE_INSTALL_ACTION += ginstall -d $(PROTO_DIR)$(USRBINDIR);
+COMPONENT_PRE_INSTALL_ACTION += ginstall -d $(PROTO_DIR)$(USRLIBDIR)/$(call tolower, $(COMPONENT_NAME))/resources
+COMPONENT_PRE_INSTALL_ACTION +=  ginstall -d $(PROTO_DIR)$(USRSHAREDOCDIR)/$(call tolower, $(COMPONENT_NAME))
+COMPONENT_INSTALL_ACTION = \
+	ginstall -m 755 $(@D)/unix/librecad $(PROTO_DIR)$(USRBINDIR); \
+	  ginstall -m 755 $(@D)/unix/ttf2lff $(PROTO_DIR)$(USRBINDIR); \
+          gcp -rf  $(@D)/unix/resources/* \
+                  $(PROTO_DIR)$(USRLIBDIR)/$(call tolower, $(COMPONENT_NAME))/resources; \
+          gcp -rf  $(@D)/librecad/support/* \
+                  $(PROTO_DIR)$(USRLIBDIR)/$(call tolower, $(COMPONENT_NAME))/resources; \
+	  cd $(PROTO_DIR)$(USRLIBDIR)/$(call tolower, $(COMPONENT_NAME))/resources/plugins && mkdir 64 && mv lib*so 64; \
+	  cd $(PROTO_DIR)$(USRLIBDIR)/$(call tolower, $(COMPONENT_NAME))/resources/ && mv fonts $(PROTO_DIR)$(USRSHAREDOCDIR)/$(call tolower, $(COMPONENT_NAME)); \
+          cd $(SOURCE_DIR) && cp -p  LICENSE* gpl-* license* \
+		$(PROTO_DIR)$(USRSHAREDOCDIR)/$(call tolower, $(COMPONENT_NAME))
+	
+
+update-publish:
+	@echo "target update-publish"
+	@[ -d $(SOURCE_DIR) ] || \
+	$(GIT) clone -b $(GIT_BRANCH) $(GIT_REPO) $(SOURCE_DIR)
+	cd $(SOURCE_DIR) ; $(GIT) pull $(GIT_REPO) ; \
+	  [ "$$($(GIT) log -1 --format=%H)" == "$$(cat .downloaded)" ] || \
+	  ( $(GIT) log -1 --format=%H > .downloaded && $(MAKE) -C $$OLDPWD publish )
+
+$(SOURCE_DIR)/.prep: $(SOURCE_DIR)/.downloaded Makefile
+
+COMPONENT_PRE_BUILD_ACTION = \
+        (cd $(@D) && qmake $(call tolower,$(COMPONENT_NAME)).pro)
+
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+REQUIRED_PACKAGES += library/qt4
+REQUIRED_PACKAGES += library/zlib
+REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/library/g++-4-runtime
+REQUIRED_PACKAGES += system/library/security/libgcrypt
+REQUIRED_PACKAGES += x11/library/libx11
+REQUIRED_PACKAGES += x11/library/libxi
+REQUIRED_PACKAGES += x11/library/libxtst
+REQUIRED_PACKAGES += library/boost
+REQUIRED_PACKAGES += library/muparser
+
+include ../../../make-rules/depend.mk
+
+BUILD_PKG_DEPENDENCIES += library/boost
```

Comment:
depend.mk and BUILD_PKG_DEPENDENCIES are not needed. If packages are required to build librecad, they should be added to REQUIRED_PACKAGES